lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -342,7 +342,8 @@ export class ResponseHandler {
         idTokenObj?: AuthToken,
         requestState?: RequestStateObject,
         code?: string,
```

Comment:
Change this to `serverTokenResponse` instead of passing in code and spa_accountId separately?

Reply:
Done